ts/smelter-web-wasm/src/react/hooks/useSmelter.ts

```diff
@@ -18,10 +19,15 @@ export function useSmelter(): Smelter | undefined {
     return () => {
       cancel = true;
       void (async () => {
-        await promise.catch(() => {});
+        await promise.catch(console.error);
```

Comment:
think about some alternative how this can be handled, is there somthing use can to to handle the error?